services/gps2tsdb/gps.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/python3
+
+# DBUS imports
+from gi.repository import GLib
+from dbus.mainloop.glib import DBusGMainLoop
+import dbus
+
+# Postgres imports
+import postgres
+
+# Environmental variable reading
+import os.environ
+
+
+# Process GPS data, called every time a fix is acquired
+def fix(*args):
+    # Data comes in as an array, extract the relevant data. See the DBUS section of `man 8 gpsd` for more information
+    time = float(args[0])
+    lat = float(args[3])
+    lng = float(args[4])
+
+    # Print it all out for debugging
+    print("Time: ", time, "\tLat: ", lat, "\tLng: ", lng)
```

Comment:
We need to use some sort of ENV based logger (like `debug` for node.js). Everything that makes it to stdout is logged to the disk by docker.

Reply:
Will do. Also should decrease the verbosity of the logging as well. 3+ lines per point is a bit much